nofish/nofish.cpp

```diff
@@ -80,14 +81,24 @@ int Main_IsMIDIGridDotted(COMMAND_T* = NULL)
 	return toggleState;
 }
 
+int Main_IsMIDIGridSwing(COMMAND_T* = NULL)
```

Comment:
This can be simplified a little:

```cpp
// Grid: Set grid type to straight/triplet/dotted
for (int cmd = 41003; cmd <= 41005; ++cmd) {
	if (GetToggleCommandStateEx(ME_SECTION, cmd))
		return 0;
}

return 1;
```

Reply:
Some MIDI editor actions for grid report state and some do not, it's best to leave it verbose to know what's going on.

Reply:
I see, good reason. Before I noticed all three `GetToggleCommandStateEx` could be merged into one, my original idea was to suggest this equivalent code to avoid doing `if(1) return 1; else return 0;`:

```cpp
return GetToggleCommandStateEx(ME_SECTION, 41005) == 0 && // Grid: Set grid type to dotted
       GetToggleCommandStateEx(ME_SECTION, 41004) == 0 && // Grid: Set grid type to triplet
       GetToggleCommandStateEx(ME_SECTION, 41003) == 0;   // Grid: Set grid type to straight
```

(...or alternatively, as you mentioned preferring to skip braces when possible–which I also do, each branch is 1 line so the braces can be removed here.)

Reply:
I really don't mind if someone knowledge updates my code, so please, never hesitate to do so!